pylib/cqlshlib/cql3handling.py

```diff
@@ -1765,6 +1768,14 @@ def drop_trigger_completer(ctxt, cass):
     return list(map(maybe_escape_name, names))
 
 
+syntax_rules += r'''
+<addIdentityStatement> ::= "ADD" "USER" ("IF" "NOT" "EXISTS")? <stringLiteral>
```

Comment:
the statement is `ADD IDENTITY IF NOT EXISTS '<identity>' TO ROLE '<role>'` similarly for drop statement.